Handle unsorted glob in the tests

mike-burns opened the pull request:
The `Dir.glob` method produces files in an OS-dependent sorting order;
in fact, the order is undefined. Use the `#contain_exactly` matcher
instead of `#eq` to handle this situation.

mike-burns committed:
Handle unsorted glob in the tests

The `Dir.glob` method produces files in an OS-dependent sorting order;
in fact, the order is undefined. Use the `#contain_exactly` matcher
instead of `#eq` to handle this situation.

afeld commented on spec/generated_site_spec.rb:
```diff
@@ -39,7 +39,8 @@ module JekyllPagesApi
         end
-        expect(Dir.glob(File.join(content_dir, '**', '*'))).to eq(paths)
+        expect(Dir.glob(File.join(content_dir, '**', '*'))).to contain_exactly(
+          *paths)
```
Good catch! What would you think about using [`match_array`](https://www.relishapp.com/rspec/rspec-expectations/v/3-5/docs/built-in-matchers/contain-exactly-matcher) instead? Makes it _slightly_ cleaner since you don't need to have the splat (`*`).

mike-burns replied:
Great idea!

mike-burns committed:
Use match_array instead of contain_exactly

This avoids a splat and is shorter, which hopefully means it will fit
under 80 characters too.
